A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_Init.js

```diff
@@ -0,0 +1,377 @@
+class CDUAocInit {
+    static ShowPage(mcdu) {
+        mcdu.clearDisplay();
+        mcdu.page.Current = mcdu.page.AOCInit;
+        mcdu.activeSystem = 'ATSU';
+
+        let fltNbr = '_______[color]red';
+        let originIcao = '____[color]red';
+        let destinationIcao = '____[color]red';
+        let ete = "____[color]red";
+        let fob = `{small}---.-{end}[color]white`;
+
+        const currentFob = formatWeight(SimVar.GetSimVarValue("FUEL TOTAL QUANTITY WEIGHT", "kg"));
+
+        function formatWeight(value) {
+            return (value / 1000).toFixed(1);
+        }
```

Comment:
please use mcdu.getFOB() instead of this. FMCMainDisplay contains quite a number of new methods to retrieve fuel information as done in #1524. https://github.com/flybywiresim/a32nx/blob/58ecd2b5844d5f4263f0c86b87c5e3652819df82/A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js#L1901-L1907 Also if you don't ever find a method that may be useful to you and plan to create it, do it in FMCMainDisplay so it can be accessed in other areas.

Reply:
Sure! Theres a specific reason why you convert pound to ton manually, instead of using "kg" (NumberVarUnit) as a second argument for the GetSimVarValue method?

Reply:
it's just the unit type I read on the simvar documentation, honestly I didn't know you can request a different numberVarUnit